sentry-spring-boot-starter/build.gradle.kts

```diff
@@ -0,0 +1,117 @@
+import com.novoda.gradle.release.PublishExtension
+
+plugins {
+    `java-library`
+    kotlin("jvm")
+    jacoco
+    id(Config.QualityPlugins.errorProne)
+    id(Config.Deploy.novodaBintray)
+    id(Config.QualityPlugins.gradleVersions)
+    id(Config.BuildPlugins.buildConfig) version Config.BuildPlugins.buildConfigVersion
+    id(Config.BuildPlugins.springBoot) version Config.springBootVersion apply false
+}
+
+apply(plugin = Config.BuildPlugins.springDependencyManagement)
+
+the<io.spring.gradle.dependencymanagement.dsl.DependencyManagementExtension>().apply {
+    imports {
+        mavenBom(org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES)
+    }
+}
+
+configure<JavaPluginConvention> {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach {
```

Comment:
it'd be nice to import `org.jetbrains.kotlin.gradle.tasks.KotlinCompile` on the top of the file and just use `KotlinCompile`, same for other full package classes, just a nit though.